compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/desugar/AnnotationDesugar.java

```diff
@@ -138,6 +251,9 @@ public class AnnotationDesugar {
     private final Names names;
     private SymbolResolver symResolver;
     private ConstantValueResolver constantValueResolver;
+    private BLangNode result;
+    private SymbolEnv env;
+    private Queue<BLangSimpleVariableDef> queue;
```

Comment:
What's this for?

Reply:
Removed this phase. please check https://github.com/ballerina-platform/ballerina-lang/pull/38488/commits/ebc4c182821564e618d7a3a9233b1aa933a0783f